pkg/model/dev.go

```diff
@@ -79,6 +79,10 @@ const (
 	OktetoRestartAnnotation = "dev.okteto.com/restart"
 	//OktetoStignoreAnnotation indicates the hash of the stignore files to force redeployment
 	OktetoStignoreAnnotation = "dev.okteto.com/stignore"
+	//OktetoDivertLabel indicates the object is a diverted version
+	OktetoDivertLabel = "dev.okteto.com/divert"
+	//OktetoDivertServiceModificationAnnotation indicates the service modification done by diverting a service
+	OktetoDivertServiceModificationAnnotation = "divert.okteto.com/modification"
```

Comment:
can you think of a way to make sure that this are consistent between cloud and cli?

Reply:
I've created an issue for this. I don't know exactly which code will end up in each spot but some of these things are shared by the cli and the operator and will end up in the weaver repo and imported as a dep into cli. There is another set of code that builds on top of that shared code and makes additions to it that is just for cli. 

I don't think it needs to happen for the first iteration but it needs to be addressed before we build more on top of it or it will get harder to fix. https://github.com/okteto/app/issues/2333